src/dnx.windows/dnx.exe.manifest

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
+    <!-- Compatibility manifests so we get the right behavior out of APIs -->
+    <!-- See: https://msdn.microsoft.com/en-us/library/windows/desktop/dn481241(v=vs.85).aspx -->
+    <compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1"> 
+        <application> 
+            <!-- Windows 10 --> 
```

Comment:
Adding the manifest will change how we treat dnx on windows 10 - before it was a win32 app and now it is onecore because of [this check](https://github.com/aspnet/dnx/blob/dev/src/dnx.windows/dnx.cpp#L16). I think it should be fine. One caveat - people who don't have Windows10 SDK installed won't be able to run dnx they built on their boxes if they are on Windows 10 since they wont have dnx.onecore.dll.